gnosis/safe-react
# Set recipient when going back from review

**What**:

Closes #2206 

**Why**:

When sending funds, if you went from the review page to the sending form, the reciever input got cleaned

**How**:

When going back to the Send Funds component, the reciever input gets setted with corresponent address

**How to test it**:

Open send funds
Copy paste a valid address that is NOT in the address book or is an owner of the safe. (you can use a QR code as well)
Fill the rest of the form with any values
Go to the review step
Go back to the form

Result:

The recipient input should have the address previously added

**Checklist**:
